manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GNU.TeXmacs
+PackageVersion: 2.1.2
+PackageLocale: en-US
+Publisher: GNU
+PackageName: TeXmacs

```

```suggestion
PackageName: Texmacs
```
Must match control panel